src/libs/security-hub-lib.ts

```diff
@@ -5,14 +5,31 @@ import {
   GetFindingsCommandOutput,
   Remediation,
   AwsSecurityFinding,
-  Resource,
   AwsSecurityFindingFilters
 } from '@aws-sdk/client-securityhub'
 import {SecurityHubJiraSyncConfig} from '../macfc-security-hub-sync'
 import {extractErrorMessage} from '../index'
 
+export interface Resource {
+  Type: string | undefined
+  /**
+   * @public
+   * <p>The canonical identifier for the given resource type.</p>
+   */
+  Id: string | undefined
+  /**
+   * @public
+   * <p>The canonical Amazon Web Services partition name that the Region is assigned to.</p>
+   */
```

Comment:
Same as above? Since the other variables do not have comments, it might be cleaner to remove them.

Reply:
removed